core/templates/pages/footer_js_libs.html

```diff
@@ -6,12 +6,12 @@
   <% if (webpackConfig.mode == 'production') { %>
     <script src="/build/webpack_bundles/<%= htmlWebpackPlugin.files.js[chunk] %>"></script>
   <% } else { %>
-    <script src="/webpack_bundles/<%= htmlWebpackPlugin.files.js[chunk] %>"></script>
+    <script src="<%= htmlWebpackPlugin.files.js[chunk] %>"></script>
   <% } %>
 <% } %>
 
-<% if (webpackConfig.mode == 'production') { %>
+<% if (htmlWebpackPlugin.options.hybrid && webpackConfig.mode == 'production') { %>
```

Comment:
Added a flag for the hybrid pages.
With this flag, angular js and third_party (angularjs libraries) scripts are only included in the hybrid pages.